gui/src/2D/controls/selector.ts

```diff
@@ -0,0 +1,528 @@
+import {Rectangle} from "./rectangle";
+import {StackPanel} from "./stackPanel";
+import {Control} from "./control";
+import {TextBlock} from "./textBlock";
+import {Checkbox} from "./checkbox";
+import {RadioButton} from "./radioButton";
+import {Slider} from "./slider";
+
+/** Class used to create a SelectorGroup 
+ * which contains groups of checkboxes, radio buttons and sliders
+*/
+export class SelectorGroup {
+    private _selectNb = 0;
+    private _groupPanel = new StackPanel();
+    private _selectors: StackPanel[] = new Array();
+    private _sliders: Slider[] = new Array();
+    
+
+    /**
+     * Creates a new SelectorGroup
+     * @param name of group, used as a group heading
+     * @param type specifies a check box, radio button or slider grouping
+     */
+    constructor(public name: string, public type: string) {
+        if (type === void 0) { type = "C"; }
+        type = type.substr(0,1).toUpperCase();
+        if(type !=="R") {

```

Comment:
why not if (type !== "R" && ...)?

Reply:
No good reason have update this. By the way I had not realised you could see the changes when I had pushed to my repository and not submitted the PR so had not bothered to check until now after my PR earlier today hence why I had not made any changes.

Reply:
No worry :)